### PR TITLE
Logo connect homepage

### PR DESCRIPTION
**Pull request by Gdelmarl0l**
added logo and link from logo to index (can change to homepage later)
added some css on navbar and changed background color

### CODE REVIEW

File: app/assets/stylesheets/components/_card.scss [hunk 1]
```diff
@@ -69,22 +69,19 @@
...
 }
 
 .practical-info-card {
   display: flex;
   justify-content: space-between;
   padding: 8px;
-  border-bottom: #606D5D 1px;
-  border-top: #606D5D 1px;
+  border-bottom: #444F5A 1px;
+  border-top: #444F5A 1px;
```
Comment (Thms25) [inline app/assets/stylesheets/components/_card.scss:86]: 1px solid color :)

File: app/javascript/controllers/drop_down_log_controller.js [hunk 2]
```diff
@@ -8,7 +8,6 @@ export default class extends Controller {
   static targets = ["menu"]
 
   dropdown() {
-    console.log('hello');
```
Comment (Thms25) [inline app/javascript/controllers/drop_down_log_controller.js:11]: merci

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -1,17 +1,15 @@
-<div class="nav-section" data-controller="drop-down-log">
```
Comment (Thms25) [inline app/views/shared/_navbar.html.erb:1]: put this back !

File: app/assets/stylesheets/components/_card.scss (hunk 1 above)
Comment (Gdelmarl0l) [reply app/assets/stylesheets/components/_card.scss:86]: done

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (Gdelmarl0l) [reply app/views/shared/_navbar.html.erb:1]: done

**Pull request merged**